docs/docs-source/docs/modules/run-cloudflow-gcp-marketplace/pages/run-sample-application.adoc

```diff
@@ -0,0 +1,102 @@
+= Run a Sample Cloudflow Application on GCP
+:toc:
+:toc-title: ON THIS PAGE
+:toclevels: 2
+
+== Setting up the Cloudflow `kubectl` Plugin
+
+With Cloudflow installed on a GKE cluster through the Marketplace, you can interact with Cloudflow via its `kubectl` plugin: `kubectl cloudflow`.
+If you haven't already, download and set up the `kubectl` plugin following the https://cloudflow.io/docs/current/get-started/prepare-development-environment.html#_download_and_install_the_cloudflow_cli[instructions].
+
+Then connect your local `kubectl` to the GKE cluster on which Cloudflow is installed. You can find the command to do that from the "Clusters" view:
+
+.Connect `kubectl` to GKE cluster
+image::connect-clusters.png[Connect kubectl to GKE cluster]
+
+It should look like the following:
+[source,bash]
+----
+$ gcloud container clusters get-credentials <cluster-name> --zone <zone> --project <gcp-project>
+----
+
+== Building and Deploying call-record-aggregator
+
+Next we'll deploy a sample Cloudflow application called https://github.com/lightbend/cloudflow/tree/master/examples/call-record-aggregator[call-record-aggregator] to the cluster.
+Clone the Cloudflow https://github.com/lightbend/cloudflow[Github repo] and `cd` into the `examples/call-record-aggregator` directory. Then run the following command to build the application:
+
+[source,bash]
+----
+$ sbt buildApp
+----
+
+Once it's complete, you should see at the end of the output a `kubectl cloudflow deploy` command to deploy your application. Run it:
+
+[source,bash]
+----
+$ kubectl cloudflow deploy call-record-aggregator.json
```

Comment:
This command failed with:
```
jamielongmuir@Jamies-MBP16 call-record-aggregator % kubectl cloudflow deploy /Users/jamielongmuir/code/lightbend/cloudflow/examples/call-record-aggregator/target/call-record-aggregator.json

[Error] This version of kubectl Cloudflow is not compatible with the Cloudflow operator, please upgrade the Cloudflow operator
```
As I just installed Cloudflow on GCP, upgrading the operator doesn't feel like the right solution... is it kubectl I should upgrade? The Cloudflow CLI? 

It'd be helpful if this message included the versions, though that can wait for a later update.

Reply:
It's the Cloudflow kubectl plugin (`kubectl-cloudflow`) that you should upgrade. The error message is indeed misleading. I created https://github.com/lightbend/cloudflow/issues/531 to track it.

Reply:
Upgraded the CLI... docs on that could be a lot better, but we can fix it after all this. I created this issue: https://github.com/lightbend/cloudflow/issues/532

Reply:
I think we should also explain what this command does - as it looks like it's trying to do multiple things. 

Once the command ran, it asks for a username for the docker registry. I'm not sure where to find that, but will look into that later. I assume that's in the GCR docs, but we should give the user a heads up that it will ask for username / password, and where to find that - even if just pointing to another doc.

Reply:
I added a note about this. The credentials are for the Docker registry hosting the application images, as configured in `target-env.sbt`